pages/doc/opentracing_to_opentelemetry_migration.md

```diff
@@ -173,4 +180,8 @@ Save the changes and run the application to send the trace data to Wavefront.
 
 ## Next Steps
 
+<<<<<<< HEAD
```

Comment:
Please resolve the conflict and instead of "Tanzu Observability", use "our service".